Apps/frontend/cypress/e2e/2-control-panel/distribute-button.cy.js

```diff
@@ -0,0 +1,12 @@
+/// <reference types="cypress" />
+
+describe("everything visible on front page", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:5173/");
+  });
+
+  it("click distribute wave button", () => {
+    // We use the `cy.get()` to get the canvas element
+    cy.get("#distribute-button").click();
```

Comment:
Personal preference, don't use `id`'s for testing, rather introduce `data-cy`.

Reply:
Because the tests are working right now and other components are tests in a similar way, i would keep it like this for this issue (pretty bit and of scope already). And unify it in the next sprint.